### PR TITLE
Add 6-str. Electric Bass and 5-str. Electric Bass (High C) plus Tablature

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
plus Tablature. And also Baritone Guitar. See https://musescore.org/en/node/269344

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
May need to add a tablature version for this too

**Comment by lasconic** [comment]
See https://musescore.org/en/node/269344#comment-819921

**Comment by Jojo-Schmitz** [comment]
See https://musescore.org/en/node/269344#comment-819874 ;-)

**Comment by mgavioli** [comment]
I don't understand the need to add a new tab style: there are already (or at least there were in ver. 2) three 6-string tab styles. Am I missing something?

**Comment by lasconic** [comment]
It's not about tab style. The ones we have are fine.  it's about ready made instruments... Users are lazy and expect to see the instruments they want/own in the list. So it would be good to have ready made 5 and 6 strings bass with the most common tuning.

**Comment by mgavioli** [comment]
Ok, I see, thanks for the additional details.

**Comment by lasconic** [comment]
No pb. Thank you for being around carefully checking PRs !

File: share/instruments/instruments.xml [hunk 1]
```diff
@@ -9967,6 +9989,85 @@
 <!--                  <clef>TAB</clef> -->
                   <genre>common</genre>
             </Instrument>
+            <Instrument id="5-string-electric-bass-high-c">
+                  <longName>5-str. Electric Bass (High C/Tenor)</longName>
```
Comment (lasconic) [inline share/instruments/instruments.xml]: What about using this string for the trackname and make longname shorter ? "5-str. Electric Bass (High C/Tenor)" looks quite long on a score...

Comment (Jojo-Schmitz) [reply share/instruments/instruments.xml]: Hope I got it right now

**Comment by Jojo-Schmitz** [comment]
AppVeyor failed for reasons unrelated to this PR (I believe)

**Comment by Jojo-Schmitz** [comment]
Yet again a Travis failure that is unrelated to the changes in this PR

**Pull request merged**